### ISSUE TITLE
Summon Dragon Issue

### ISSUE THREAD

**Issue by Crusufix** [MrVauxs/pf2e-jb2a-macros#83]
When my level 9 sorcerer player uses Summon Dragon to summon a Level 5 Fire Drake the token appears with missing stats as below. 

![image](https://user-images.githubusercontent.com/95157857/210049095-bb4857c0-918b-49fc-a74e-8fb815fbf8d9.png)

This appears to only happen on the level 5 Fire Drake, all the other summoned tokens appear normal. This also appears to only happen when my player uses the spell, if I do it as the GM the Fire Drake token appears normal.

Thank you for looking into this.

**Comment by MrVauxs** [comment]
Cannot replicate. Make sure you are up to date on everything.
https://user-images.githubusercontent.com/32039708/210090143-5706db53-1791-4577-96b9-2abd814b2e41.mp4

**Comment by Crusufix** [comment]
Hello, I've done more troubleshooting on this and I've noticed a strange interaction. If my player summons the drake within our Marshalls Aura, then this bug happens. If he summons outside the aura, the fire drake summons in fine. Strangely it still only seems to affect the Fire Drake?

**Comment by MrVauxs** [comment]
Still can't replicate the issue. Make sure you are on newest version of the system as well.

**Comment by Crusufix** [comment]
I was on the latest release of 2.8.5 and I did double-check my Flame Drake and it's stats were good. I think I've resolved my problem.. and it's definitely a weird one. The Sorcerer in question appeared to have a bad Battle Cry feat. Whenever he summoned the Flame drake the console would always show a bunch of errors, but this was the same for all the other summons, except one. For some reason summoning the Flame drake in the Marshall's aura popped this error message in the console. "PF2e System | Note rules element on item Battle Cry (Actor..Item.6OxfgMEdkZjBLJW8) failed to validate: "outcome" property is missing or invalid"
I decided to remove his Battle Cry feat, and the summon worked perfectly.

I re-added the Battle Cry feat and that error message is no longer popping up in the Console and the summon is working perfectly.
Checking on his characters backup, I do see that the Battle Cry feat has changed recently.
![image](https://user-images.githubusercontent.com/95157857/210127601-8813e459-6f10-455c-997b-e4289b4dbfd0.png)
![image](https://user-images.githubusercontent.com/95157857/210127604-614bd2e7-8c68-47cd-bcfe-f71f9409e6d4.png)
So, yeah... weird.. but I think it's resolved now.
Just to make sure. I imported his Backup character, tested the summon and I could replicate the error. Swapped the Battle Cry to the updated one and error doesn't happen.
*SHRUG*
